VirtoCommerce.Storefront/Controllers/Api/ApiListsController.cs

```diff
@@ -107,15 +95,97 @@ public class ApiListsController : StorefrontControllerBase
             }
         }
 
+        // POST: storefrontapi/lists/search
+        [HttpPost]
+        public async Task<ActionResult> SearchLists([FromBody] ShoppingCartSearchCriteria searchCriteria)
+        {
+            if (searchCriteria == null)
+            {
+                searchCriteria = new ShoppingCartSearchCriteria();
+            }
+
+            //restricting query to lists search, only paging and sorting are available
+            searchCriteria.Type = "wishlist";
```

Comment:
@tatarincev Would it be better to use "wishlist" as default parameter of search criteria, instead of hard coding? Possible will allow us to use multiple types of lists (i.e. wishlist, recipe, something else...) without changing code.